lightbulb/decorators.py

```diff
@@ -98,13 +98,17 @@ def command(
         inherit_checks (:obj:`bool`): Whether or not the command should inherit checks from the parent group. Only
             affects subcommands. Defaults to ``False``.
         pass_options (:obj:`bool`): Whether the command will have its options passed as keyword arguments when invoked.
+        name_localizations (Mapping[Union[hikari.locales.Locale, :obj:`str`], :obj:`s®tr`]): A set of name localizations for this command. Defaults to an empty dictionary.
```

Comment:
```suggestion
        name_localizations (Mapping[Union[hikari.locales.Locale, :obj:`str`], :obj:`str`]): A set of name localizations for this command. Defaults to an empty dictionary.
```

Reply:
Fixed in c5888c1b1d9e40f58b7d092bb3f5aa454968c7e7.